pubsub/azure/servicebus/servicebus.go

```diff
@@ -36,38 +36,7 @@ import (
 )
 
 const (
-	// Keys.
```

Comment:
These defaults were just moved to the `metadata.go` file so they're together with the emtadata keys